### ISSUE TITLE
Basic.t failed "test_cli died" on Strawberry Perl

### ISSUE THREAD

**Issue by wchristian** [houseabsolute/perl-code-tidyall#72]
Getting this test failure on Strawberry Perl:

```
C:\Users\Mithaldu>perl -v

This is perl 5, version 18, subversion 4 (v5.18.4) built for MSWin32-x86-multi-thread-64int

[...]

# Versions for all modules listed in MYMETA.json (including optional ones):
#
# === Configure Requires ===
#
#     Module              Want Have
#     ------------------- ---- ----
#     ExtUtils::MakeMaker  any 7.24
#
# === Build Requires ===
#
#     Module              Want Have
#     ------------------- ---- ----
#     ExtUtils::MakeMaker  any 7.24
#
# === Test Requires ===
#
#     Module              Want     Have
#     ------------------- ---- --------
#     Encode               any     2.86
#     ExtUtils::MakeMaker  any     7.24
#     File::Spec           any     3.62
#     Test::Class::Most    any     0.08
#     Test::Differences    any     0.64
#     Test::More          0.96 1.302062
#     Test::Warnings       any    0.026
#     autodie              any     2.29
#     lib                  any     0.63
#
# === Test Recommends ===
#
#     Module         Want     Have
#     ---------- -------- --------
#     CPAN::Meta 2.120900 2.150010
#
# === Runtime Requires ===
#
#     Module                          Want     Have
#     --------------------------- -------- --------
#     Capture::Tiny                    any     0.44
#     Config::INI::Reader              any    0.025
#     Cwd                              any     3.62
#     Data::Dumper                     any    2.161
#     Date::Format                     any     2.24
#     Digest::SHA                      any     5.96
#     Exporter                         any     5.72
#     File::Basename                   any     2.84
#     File::Find                       any     1.23
#     File::Path                       any     2.12
#     File::Slurp::Tiny                any    0.004
#     File::Spec                       any     3.62
#     File::Which                      any     1.21
#     File::Zglob                      any     0.11
#     Getopt::Long                     any     2.49
#     Guard                            any    1.023
#     IPC::Run3                        any    0.048
#     IPC::System::Simple              any     1.25
#     List::Compare                    any     0.53
#     List::SomeUtils                  any     0.53
#     Log::Any                         any    1.042
#     Moo                         2.000000 2.002004
#     Moo::Role                        any 2.002004
#     Path::Tiny                     0.098    0.098
#     Scalar::Util                     any     1.46
#     Specio                          0.30     0.30
#     Specio::Declare                  any     0.30
#     Specio::Library::Builtins        any     0.30
#     Specio::Library::Numeric         any     0.30
#     Specio::Library::Path::Tiny     0.04     0.04
#     Specio::Library::String          any     0.30
#     Test::Builder                    any 1.302062
#     Text::Diff                      1.44     1.44
#     Text::Diff::Table                any     1.44
#     Text::ParseWords                 any     3.30
#     Time::Duration::Parse            any     0.13
#     Try::Tiny                        any     0.27
#     base                             any     2.23
#     constant                         any     1.33
#     strict                           any     1.07
#     warnings                         any     1.18
#
# === Runtime Recommends ===
#
#     Module                Want Have
#     --------------------- ---- ----
#     Parallel::ForkManager  any 1.17
#
t/00-report-prereqs.t .. ok
t/Basic.t .............. 137/?
#   Failed test 'test_cli died (List form of pipe open not implemented at t/lib/Test/Code/TidyAll/Basic.pm line 535.)'
#   at t/Basic.t line 4.
#   (in Test::Code::TidyAll::Basic->test_cli)

#   Failed test 'test filemode'
#   at t/lib/Test/Code/TidyAll/Basic.pm line 92.
#   (in Test::Code::TidyAll::Basic->test_filemode)
#          got: '33206'
#     expected: '33261'
t/Basic.t .............. 167/? # Looks like you failed 2 tests of 187.
t/Basic.t .............. Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/187 subtests
t/Conf.t ............... ok
t/Plugin-SortLines.t ... ok
t/Zglob.t .............. ok

Test Summary Report
-------------------
t/Basic.t            (Wstat: 512 Tests: 187 Failed: 2)
  Failed tests:  138, 160
  Non-zero exit status: 2
Files=5, Tests=210, 10 wallclock secs ( 0.08 usr +  0.08 sys =  0.16 CPU)
```

**Comment by autarch** [comment]
Does this just need to skip this test on some platforms? Alternatively we could maybe use `IPC::Run3` to call this command and capture the output, if that works on Win32 (I think it does).

**Comment by autarch** [comment]
Fixed in 0.56.
